### ISSUE TITLE
[Frontend] Make navbar sticky on privacy policy page

### ISSUE THREAD

**Issue by Ram81** [Cloud-CV/EvalAI#3675]
### Description

Currently, the navbar on the privacy policy page is not stick on large screens. As part of fix of this issue we need to make the navbar sticky on all resolutions of the screen. See the screenshot for reference of current issue.

![Screenshot from 2022-01-17 17-57-55 (1)](https://user-images.githubusercontent.com/16323427/149845661-c216dd18-6c83-487d-b3de-a092425c0c44.png)

**Comment by Ram81** [comment]
Hi @Ayush-Raghuwanshi, sure you can work on this issue. Let us know if you have any questions

**Comment by cypersii** [comment]
Hey, can i contriburte to this issue?

**Comment by Suryansh5545** [comment]
Seems like Ayush isn't working on this issue so can i take it?

**Comment by Ayush-Raghuwanshi** [comment]
Yeah I am so sorry, @Ram81 you can assign other. I cant work on it right now, I am sorry...

**Comment by gchhablani** [comment]
@Ram81 @RishabhJain2018 Does this issue/PR look relevant? I think the PR fixes the issue.

**Comment by Mo7ammad-safwat** [comment]
Hi I am keenly interested in addressing specific issues and am earnestly hopeful of having the opportunity to engage with them further.

**Comment by RishabhJain2018** [comment]
Closing this since we are not working on Frontend_v2
